README.md

```diff
@@ -50,3 +51,11 @@ Test your new function; I've provided some basic testing code in `code.test.js`.
 What is the worst-case asymptotic time complexity of your implementation? What
 is the worst-case asymptotic memory complexity? Add your answer, including your
 reasoning, to this markdown file.
+
+**MY ANSWER:**
+
+If I am correct, I believe that the number of possible swaps there could be is $n^2$, which should mean that the outer while loop could run for about $n^2$ iterations. Within this while loop, we have 2 nested loops that iterate over all the elements in the currentRoute array. So this part of our code gives us the expression $n^2$. So we have $n^2$ iterations of $n^2$...
```

Comment:
Your stopping criterion looks at whether a solution improves -- this is what determines when you'll stop. The number of swaps is irrelevant.